lizmap/tooltip.py

```diff
@@ -287,8 +287,10 @@ def _generate_external_resource(widget_config, name, fname):
             field_view = '''
                     concat(
                         '<a href="',
-                        "{}",
-                        '" target="_blank">{}</a>'
+                        "{0}",
+                        '" target="_blank">',
+                        base_file_name({0}),
+                        '</a>'
                     )'''.format(name, fname)
```

Comment:
The variable `fname` is not used.

This will make Lizmap incompatible with QGIS Server < 3.8, `base_file_name` has been introduced in 3.8.
Maybe keep the previous behavior if < QGIS 3.8 ?